### PR TITLE
Added a prettier setup

### PR DESCRIPTION
**Pull request by sunesimonsen**
This is a the same kind of setup we have in unexpected.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,32 +3,25 @@
   "version": "4.0.0",
   "description": "DOM plugin for the unexpected assertion libary",
   "main": "lib/index.js",
-  "files": [
-    "lib",
-    "unexpected-dom.js",
-    "unexpected-dom.min.js"
-  ],
+  "files": ["lib", "unexpected-dom.js", "unexpected-dom.min.js"],
```
Comment (papandreou) [inline package.json]: Could we add `.prettierignore` and add `package.json` to it? In my experience, `prettier` will battle `npm install --save`, changing it back and forth.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    'import/no-extraneous-dependencies': [
+      'error',
+      {
+        devDependencies: ['**/test/**/*.js'],
+        optionalDependencies: false,
+        peerDependencies: false
+      }
+    ]
+  }
+};
```
Comment (papandreou) [inline .eslintrc.js]: How about [@gustavnikolaj's `isTTY` trick](https://github.com/unexpectedjs/unexpected/blob/a40ec480bbc3259a20c0cdcb5df05b2b6baac83e/.eslintrc.js#L17-L23)?

Comment (sunesimonsen) [reply .eslintrc.js]: Fixed it.

**Comment by sunesimonsen** [comment]
I'll just merge it as I have just addressed the feedback :-)

**Pull request merged**